python/pyspark/sql/connect/session.py

```diff
@@ -694,11 +694,11 @@ def streams(self) -> "StreamingQueryManager":
     streams.__doc__ = PySparkSession.streams.__doc__
 
     def __getattr__(self, name: str) -> Any:
-        if name in ["_jsc", "_jconf", "_jvm", "_jsparkSession"]:
+        if name in ["_jsc", "_jconf", "_jvm", "_jsparkSession", "sparkContext"]:
             raise PySparkAttributeError(
                 error_class="JVM_ATTRIBUTE_NOT_SUPPORTED", message_parameters={"attr_name": name}
             )
-        elif name in ["newSession", "sparkContext"]:
+        elif name in ["newSession"]:
```

Comment:
Let's also do `newSession`. Can we fix them all since we're here? I think we shouldn't say it's not implemented but unsupported explicitly. e.g., `DataFrame.rdd`

Reply:
Sure, will add more if I find any.